.gitignore

```diff
@@ -1,11 +1,26 @@
 ### Visual Studio ###
 
+# common
+*.suo
+*.user
+*.sln.docstates
 .vs/
+**/packages/
 
+# build results
+[Bb]in/
 [Ob]bj/
 [Dd]ebug/
 [Rr]elease/
 
-**/packages/
+# Roslyn cache directories
+*.ide/
+
+# MSTest test results
+[Tt]est[Rr]esult*/
+[Bb]uild[Ll]og.*
+
+# dnx lock.json
+project.lock.json
+*Tests/config.local.json
```

Comment:
저도 repositories.config파일이 필요한 경험은 하지 못했습니다. [공식문서](http://docs.nuget.org/Consume/Package-Restore)에서도 아래와 같이 말하는 것 보니, 현 상태로 무시해도 될 것 같습니다.

>Uncomment if necessary however generally it will be regenerated when needed
>!**/packages/repositories.config